### ISSUE TITLE
Feature request: mark a deck as favourite

### ISSUE THREAD

**Issue by feralheart** [JonathanGuberman/unmatched_maker#84]
It would be good to mark a deck as favourite or mark for later

**Comment by thearchitectofsubjectivetruth** [comment]
Yes. I second that. It would be great if you could track decks this way. And even greater if you get notified, if something has changed. That way we could download an updated version if we want to. Most of the time they are not exactly balanced yet.

If we could sign in with our own email instead of our friendly pioneers of data security discord and google, we could even get notified via email about notifications.
